src/main/java/de/tum/in/www1/artemis/domain/tutorialgroups/TutorialGroup.java

```diff
@@ -97,6 +97,14 @@ public class TutorialGroup extends DomainObject {
     @Transient
     private TutorialGroupSession nextSession;
 
+    /**
+     * This field represents the average attendance of this tutorial group
+     * <p>
+     * For more information on how this is calculated check out {@link de.tum.in.www1.artemis.service.tutorialgroups.TutorialGroupService#setAverageAttendance}
+     */
+    @Transient
+    private Integer averageAttendance;
```

Comment:
Is there a more fitting name? I am not sure what to expect with an `Integer` called `averageAttendance`. I am sure there is a reason behind this but I would expect a float/double when reading "average"